docs/dev-guide/api-ref.md

```diff
@@ -816,29 +820,25 @@ The metadata resource allows the user to export and import Channels, Clients, Me
 
 ```js
 {
-  "successes": [
+  "rows": [
```

Comment:
It seems like the `rows` property is unnecessary, we could just return an array instead.